entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 
 set -o errexit
 
+# export custom environment variables
+ENV_FILE=/srv/grafana/envs
```

Comment:
```suggestion
ENV_FILE=/srv/envvars
```
`envvars` seems to be a more canonical name